src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -98,6 +99,10 @@
     <Reference Include="System.Drawing" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xml" />
+    <ProjectReference Include="..\..\Scripting\CSharp\CSharpScripting.csproj">
+      <Project>{066f0dbd-c46c-4c20-afec-99829a172625}</Project>
+      <Name>CSharpScripting</Name>
+    </ProjectReference>
```

Comment:
This is probably fine. IncludeOutputGroupsInVSIX defaults to something which will copy the project and other stuff (like some references). The VS SDK tooling also has a list of assemblies to _never_ package (think VS SDK assemblies themselves) as a hack to prevent overly bloated VSIXes. The ForceIncludeInVSIX is a flag that overrides that, and you'll see it in some places because Roslyn is otherwise in the list for third parties.

It's entirely possible that the other properties that are there don't need to be there, but awhile back some people were horrified by all of this and made it consistent, if unnecessarily verbose.